zerver/openapi/zulip.yaml

```
@@ -9128,6 +9128,43 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: deactivate-custom-emoji
+      summary: Deactivate custom emoji
+      tags: ["server_and_organizations"]
+      description: |
+        This endpoint is used to [deactivate a custom emoji](https://zulip.com/help/custom-emoji#deactivate-custom-emoji) from the user's
+        organization.
+
+        Note: Deactivated emoji will still be visible in old messages.
+
+        **Changes**: Starting with Zulip 8.0 (feature level 190), this endpoint will now return `404` status code when the emoji does not exist instead of `400`.
```

This should use the present tense when describing the current API, so "will now return" is wrong.

---

Here's a version:

```suggestion
        **Changes**: Before Zulip 8.0 (feature level 190), this endpoint returned an HTTP status code of 400 when the emoji did not exist, instead of 404.
```

The current API is already described by what's outside the "Changes" note (if it weren't, that'd be an important thing to fix), so the "Changes" note can focus on what used to be different.